README.md

```diff
@@ -1,6 +1,131 @@
-# website
+# Website
+
 The website for the Civic Tech Index
 
+To develop the site, you'll need to first clone the repository on to your computer. For new Git users, see the [Using Git](#using-git) section below. <br><br>
+
+# OVERVIEW
+**Set up**
+1. [Join the Repo Team](#step-1-become-a-member-of-the-repository-team)
+
+2. [Using Git](#using-git) and [Fork the Repo](#step-2-fork-the-repository)
+
+3. [Clone to your local machine](#step-3-clone-your-online-repository-to-your-local-computer)
+
+4. [Set up Docker](#step-4-setting-up-docker)
+
+**Before you start working on an issue**
+
+5. [Switch to new issue branch before you start making changes](#step-5-change-to-a-new-branch)
+
+
+**After you've worked on your issue and before you make a pull request:**
+
+6. [Check upstream before you push](#step-6-check-upstream-before-you-push).
+
+6a. [No changes in the upstream repo](#step-6a-no-changes-in-the-upstream-repository)
+
+**Or**
+
+6b. [Conflicting changes in the upstream repo](#step-6b-conflicting-changes-in-the-upstream-repository) and how to resolve them
+              
+**Okay. You're good to go!**        
+ 
+7. [Complete the pull request](#step-7-complete-the-pull-request)
+
+---
+
+# Forking and cloning the repository with proper security
+
+## Step 1 Become a member of the repository Team
+
+In the `civic-tech-index` slack channel, send your GitHub name to the project manager (or on the slack channel thread) and we'll add you as a member to the GitHub repository Team.
```

Comment:
Change the order of things to be the following:
New Volunteer slack the PM their email address
PM adds to folder and slacks back roster
New Volunteer fills out roster and lets PM know they are done
New Volunteer sets up 2FA
New Volunteer is added to GitHub at next team meeting
Mark Membership public

Reply:
Addressed this, please review.